openedx.yaml

```diff
@@ -0,0 +1,11 @@
+# This file describes this Open edX repo, as described in OEP-7:
```

Comment:
change OEP-7 to OEP-2
and change the link to this
https://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification